pytorch_lightning/trainer/trainer.py

```diff
@@ -1544,13 +1544,15 @@ def _log_device_info(self) -> None:
                 " `Trainer(ipus=8)` or script `--ipus=8`."
             )
 
-    def _on_exception(self):
+    def _on_exception(self, exception):
```

Comment:
would be great to add the typing on that occasion :)

Reply:
Done !